tools/submission/submission_checker.py

```diff
@@ -3432,90 +3434,106 @@ def check_compliance_acc_dir(test_dir, model, config):
     if not os.path.exists(fname):
         log.error("%s is missing in %s", fname, test_dir)
     else:
-        # Accuracy can fail for TEST01
-        is_valid = True
-        with open(fname, "r") as f:
-            for line in f:
-                # look for: TEST PASS
-                if "TEST PASS" in line:
-                    acc_passed = True
-                    break
-        if acc_passed == False:
-            log.info(
-                "Compliance test accuracy check (deterministic mode) in %s failed",
-                test_dir,
-            )
+        if "TEST01" in test_dir:
```

Comment:
This can be problematic if someone mistakenly deletes the TEST01 compliance test (for say R50) and the submission checker can pass. The need for TEST01 check should be based on a given model.

Reply:
Hi Arjun, the test_dir check is here: https://github.com/mlcommons/inference/pull/1616/files#diff-aefddf698e77077f4f22c1ee8a2fa17ac3da94271543125e1c2b4b7e26c1f694R3591

Reply:
Thank you Zhihanj for pointing to that. Looks good to me. For 4.1, I'll try to add a github action for submission checker changes :)